provisioning/Samples/device/SymmetricKeySample/Parameters.cs

```diff
@@ -0,0 +1,59 @@
+﻿using CommandLine;
+using Microsoft.Azure.Devices.Client;
+
+namespace Microsoft.Azure.Devices.Provisioning.Client.Samples
+{
+    /// <summary>
+    /// Parameters for the application
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'i',
+            "IdScope",
+            Required = true,
+            HelpText = "The Id Scope of the DPS instance")]
+        public string IdScope { get; set; }
+
+        [Option(
+            'r',
+            "RegistrationId",
+            Required = true,
+            HelpText = "The Id of the device to register.")]
+        public string RegistrationId { get; set; }
```

Comment:
This is only a parameter in individual enrollment scenarios, FYI

Reply:
Really it ends up being the device Id. Is it more complicated than I realize?

Reply:
Okay, I see that for individual enrollments (in the Azure portal), a registration Id can be different than the device Id.

Working on it...